lib/chef/provider/windows_task.rb

```diff
@@ -51,9 +51,11 @@ def set_current_resource(task_hash)
         @current_resource.months(task_hash[:months]) if task_hash[:months]
         set_current_idle_time(task_hash[:idle_time]) if task_hash[:idle_time]
         @current_resource.random_delay(task_hash[:random_delay]) if task_hash[:random_delay]
-        @current_resource.execution_time_limit(task_hash[:execution_time_limit]) if task_hash[:execution_time_limit]
+        @current_resource.execution_time_limit(task_hash[:execution_time_limit] || "PT72H")
```

Comment:
72 hours seems long, but this is an arbitrary setting, right?

Reply:
@btm By default on windows 2008 and 2012 system `execution_time_limit` return "PT72H" and for windows 2016 it returns nil so idempotent check fails when the user executes resource without  `execution_time_limit` on windows 2016.
https://github.com/chef/chef/blob/master/lib/chef/resource/windows_task.rb#L77

Reply:
please add a comment here about this setting. Otherwise someone will come along and think 72h is too long and change it and the tests to something shorter and we might not realize until after  we release.